docs/markdown/Using Pants/advanced-target-selection.md

```diff
@@ -181,10 +181,10 @@ You can, of course, pipe multiple times:
 Sharding the input targets
 --------------------------
 
-You can leverage shell piping to partition the input targets into multiple shards. 
+The `test` goal natively supports sharding input targets into multiple shards. Use the option `--test-shard=k/N`, where k is a non-negative integer less than N. For example, you can split up your CI into three shards with `--shard=0/3`, `--shard=1/3`, and `--shard=2/3`.
 
-For example, to split your Python tests into 10 shards, and select shard 0:
+For other goals, you can leverage shell piping to partition the input targets into multiple shards. For example, to split your `lint` run into 10 shards, and select shard 0:
```

Comment:
Should we be suggesting this? For `test` we know sharding is valuable because we run one process per test file anyway, but that is not true in general, and in cases where transitive deps are needed all we'd do is compute mostly overlapping deps multiple times.

Reply:
True! Although `package` is a goal where it could be useful. What about using that, rather than `lint`?

I'm also fine with removing this paragraph.

Reply:
Yeah, package is a more straightforwardly beneficial example. Let's do that.